{
	"id": "AVG-2560",
	"advisories": [
		{
			"content": {
				"id": "ASA-202111-9",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/ASA-202111-9"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "AVG-2560",
				"severity": [
					{
						"type": "vendor",
						"source": "security.archlinux.org",
						"vendor": "High"
					}
				],
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/AVG-2560"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2021-38005",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38005"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38006",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38006"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38007",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38007"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38008",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38008"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38009",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38009"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38010",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38010"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38011",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38011"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38012",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38012"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38013",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38013"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38014",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38014"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38015",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38015"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38016",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38016"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38017",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38017"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38018",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38018"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38019",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38019"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38020",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38020"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38021",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38021"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-38022",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-38022"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "arch",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterions": [
							{
								"vulnerable": true,
								"fix_status": {
									"class": "fixed"
								},
								"package": {
									"name": "chromium"
								},
								"affected": {
									"type": "pacman",
									"range": [
										{
											"lt": "96.0.4664.45-1"
										}
									],
									"fixed": [
										"96.0.4664.45-1"
									]
								}
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "arch",
		"raws": [
			"vuls-data-raw-arch/AVG-2560.json"
		]
	}
}
